### PR TITLE
Improvement/devsu 1885 fix malformed deletions

### PR DESCRIPTION
**Pull request by dustinbleile**
For https://www.bcgsc.ca/jira/browse/DEVSU-1885

### CODE REVIEW

File: ipr/annotate.py [hunk 1]
```diff
@@ -331,7 +331,22 @@ def annotate_positional_variants(
...
+                    if (
+                        variant[-1] == '>'
+                        and 'g.' in variant
+                        and variant[-2].isalpha()
+                        and variant[-3].isnumeric()
+                    ):
+                        logger.warning(
+                            f"Assuming malformed deletion variant {variant} is {variant[:-2] + 'del'}"
+                        )
+                        variant = variant[:-2] + 'del'
```
Comment (mathieulemieux) [inline ipr/annotate.py:348]: So 'chr1:g.145402353C>' basically means 'chr1:g.145402353del' ?

File: setup.cfg [hunk 2]
```diff
@@ -18,7 +18,7 @@ known_standard_library = requests
 
 [metadata]
 name = ipr
-version = 3.7.1
+version = 3.8.0
```
Comment (mathieulemieux) [inline setup.cfg:21]: Why are we bumping the version number since we are merging to develop?

File: ipr/annotate.py (hunk 1 above)
Comment (dustinbleile) [reply ipr/annotate.py:348]: Seemed that way from the other descriptions of the event at https://www.bcgsc.ca/jira/browse/DEVSU-1885

File: setup.cfg (hunk 2 above)
Comment (dustinbleile) [reply setup.cfg:21]: Because I branched off https://github.com/bcgsc/pori_ipr_python/pull/112

And seem to be very confident it will be approved ;)

Comment (mathieulemieux) [reply setup.cfg:21]: lol, fair enough!

**Review by darenillas — APPROVED** [review]

**Review by mattdoug604 — APPROVED** [review]

**Pull request merged**